### PR TITLE
add docu for contribution to the ig

### PR DESCRIPTION
**Pull request by riedde**
This is a first draft for the contribution guidelines of the IG. It also documents the actual state of 'how to contribute to music-encoding' which is developed by the technical team and me.

To be discussed in an IG meeting.

### CODE REVIEW

**Review by doerners — CHANGES_REQUESTED** [review]
First of all, many thanks @riedde for this great work. I found some typos and made some suggestions on wording. I think the typos should be addressed before merging.

File: README.md [hunk 1]
```diff
@@ -2,14 +2,45 @@
 The repository of the Metadata Interest Group
 
 ## What is it for?
-This is a workspace where the metadata-ig store code snippets, examples and text blocs for further development. It is also a place for managing discussions, issues and so on.
+This is a workspace for the metadata-ig snippets, examples and text blocs can be stored for further development. This is also a place for managing discussions and issues the IG takes care about.
```
[the remark this answers is not in the document]
Comment (riedde) [reply README.md:5]: Thank you for this nice suggestion @doerners :)

File: README.md [hunk 2]
```diff
@@ -2,14 +2,45 @@
...
+- Pull Requests: If you like to contribute to this repository by adding some code please [open a pull request](https://github.com/music-encoding/metadata-ig/pulls).
+
+### Gitflow (IG repository)
+- Pushing directly to the directory is not recommented. Please use always the [pull request](https://github.com/music-encoding/metadata-ig/pulls) feature. This makes it easier to manage the code and improves the documentation. (E.g., if release notes are generated automatically they will only document the accepted.)
+
+### Contributing to music-encoding as metadata IG
+The workflow to contribute to the schema and guidelines is different to the workflow described above. (see also [https://github.com/orgs/music-encoding/projects/2#card-83741232](https://github.com/orgs/music-encoding/projects/2#card-83741232))
```
[the remark this answers is not in the document]
Comment (riedde) [reply README.md]: Thank you, I have improved it.

**Review by doerners — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -2,14 +2,45 @@
...
-- [...]
+There is no defined structure for this repository at the moment.
+It is recommended to organize the folders thematically.
+
+## How to contribute to the IG
+### Workflow (discussions, issues)
+
+Contribution is always welcome! There are several ways for it:
+
+- Diskussions: This should be some kind of 'save place' for everyone (newbies, old hands, etc.) to come up with everything that is on their mind: questions, ideas (that can be very vage), announcements, polls, etc. All you have to do ist to [open a discussion](https://github.com/music-encoding/metadata-ig/discussions) and you'll get feedback from the community. Every time you open a discussion please choose a category to make them easier to manage.
```
[the remark this answers is not in the document]
Comment (rettinghaus) [reply README.md:23]: @riedde Did you forget to commit this suggestion?

**Review by rettinghaus — COMMENTED** [review]
There are still some open suggestions. Please go through them again.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (riedde) [reply README.md:23]: sry, this was skipped in the GUI, so I missed it!

File: README.md [hunk 4]
```diff
@@ -2,14 +2,45 @@
...
+- Issues: If there is something the IG should work on or something that should be discussed in an IG-meeting (e.g., improvement proposals to the schema) just [open an issue](https://github.com/music-encoding/metadata-ig/issues).Every time you open an issue please label them for easier management. In comparison to a discussion an issue is something that needs work and is understand as a todo for the IG.
+- Pull Requests: If you like to contribute to this repository by adding some code please [open a pull request](https://github.com/music-encoding/metadata-ig/pulls).
+
+### Gitflow (IG repository)
+- Pushing directly to the directory is not recommented. Please use always the [pull request](https://github.com/music-encoding/metadata-ig/pulls) feature. This makes it easier to manage the code and improves the documentation. (E.g., if release notes are generated automatically they will only document the accepted.)
```
[the remark this answers is not in the document]
Comment (riedde) [reply README.md:28]: @doerners I updated your suggestion (`directory` to `repository`). Not the best way, I know, but easier to fix that right now.

**Comment by riedde** [comment]
Thank you @rettinghaus for taking care of what I'm doing! :) I resolved the open requests.

**Review by rettinghaus — APPROVED** [review]
Thank you. 
@doerners Please have a final look and merge.

**Comment by doerners** [comment]
Thanks @riedde and @rettinghaus! Looks good to me, merging this now.

**Pull request merged**